pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java

```diff
@@ -325,6 +341,31 @@ public static void main(String[] args) throws Exception {
             printAggregatedStats();
         }));
 
+        if (arguments.createTopicPartitions > 0) {
+            PulsarAdminBuilder clientBuilder = PulsarAdmin.builder()
+                    .serviceHttpUrl(arguments.adminURL)
+                    .tlsTrustCertsFilePath(arguments.tlsTrustCertsFilePath);
+
+            if (isNotBlank(arguments.authPluginClassName)) {
+                clientBuilder.authentication(arguments.authPluginClassName, arguments.authParams);
+            }
+
+            if (arguments.tlsAllowInsecureConnection != null) {
+                clientBuilder.allowTlsInsecureConnection(arguments.tlsAllowInsecureConnection);
+            }
+
+            try (PulsarAdmin client = clientBuilder.build();) {
+                for (String topic : arguments.topics) {
+                    log.info("Creating partitioned topic {} with {} partitions", topic, arguments.createTopicPartitions);
+                    try {
+                        client.topics().createPartitionedTopic(topic, arguments.createTopicPartitions);
+                    } catch (PulsarAdminException.ConflictException alreadyExists) {
+                        log.debug("Topic "+topic+" already exists: " + alreadyExists);
```

Comment:
If the topic already exists, we should check the partitions is equals to the user settings, otherwise, we might get the incorrect test result. I think we should stop the test client in this situdation.

Reply:
@codelipenghui I added the validation, PTAL
thanks for your quick review

Reply:
`log.debug` should be put in `if (log.isDebugEnabled())`

Reply:
@sijie done